### DISCUSSION TITLE
Add a "Overwrite all existing" on the overwrite dialog when exporting to multiple files

### DISCUSSION THREAD

**Discussion by ragerin** [Orama-Interactive/Pixelorama#585] [Ideas]
**Feature/enhancement description:**

When exporting to multiple files, i.e. animations as separate files, that already exist in the destination directory, a dialog prompts if the user wants to overwrite for each already existing file. This dialog requires manually clicking either "OK" or "Cancel Export".
If there are a lot of frames in the animation it takes an equal amount of clicks to overwrite each exported file.

By adding a button or checkbox to the dialog that allows to repeat the overwrite action, the user only has to click twice for an arbitrary amount of output files.

**Comment by OverloadedOrama** [comment]
![image](https://user-images.githubusercontent.com/35376950/205185689-da25e69a-6521-4728-8a60-c0609dd3f8c0.png)

This behavior has been changed in commit 5e90b740e79792a0222de738d6664cd7656ad307. Instead of showing a dialog for each existing image, it now only shows a single dialog with a list of all existing images, reducing the number of clicks required to just a single one. This change will be included in version 0.11.
